extensions/security-jpa/runtime/src/main/java/io/quarkus/security/jpa/JpaSecurityConfig.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.security.jpa;
+
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+/**
+ * A configuration object for JPA-based security.
+ */
+@ConfigRoot(name = "security.jpa", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public class JpaSecurityConfig {
```

Comment:
What is the point of the empty config? I think this should at least have the unit name.

Reply:
Yeah, I will remove the empty config. I wasn't sure if I was going to need config or not, but I can't think of anything.

Reply:
Removed